src/App.tsx

```diff
@@ -15,36 +15,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  default = '',
+  alphabet = 'Sort alphabetically',
+  length = 'Sort by length',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.default);
+  const [reversed, setReversed] = useState(false);
+
+  const visibleGoods = [...goodsFromServer];
+
+  if (sortField) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.alphabet:
+          return good1.localeCompare(good2);
+        case SortType.length:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    visibleGoods.reverse();
+  }
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={`button is-info ${sortField !== SortType.alphabet ? 'is-light' : ''}`}
```

Comment:
here you can use classnames className={cn('button is-info', {  'is-light': sortField !== SortType.alphabet } ) }